### DISCUSSION TITLE
Reduce width on Tab Form

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#5712] [Help] [answered]
My Tab Form doesn't use the full width of my screen, why ?

The Wizard and fieldset Form is ok

Thanks
![tab](https://user-images.githubusercontent.com/58070424/219629833-e86abe4f-a227-453e-9965-06c6f9d0fdf1.png)

![tab2](https://user-images.githubusercontent.com/58070424/219630141-4be27540-96b9-45ce-b4d7-ba1e0a931684.png)

**Comment by pxlrbt** [comment, marked as answer]
It would help, if you shared any code.

Probably because you are using `Tabs` inside a Form and Forms have a default of 2 columns.

`$form->columns(1)` would solve this.

**Reply by eaziweb95** [reply]
It works! Thanks a lot !
